test/controllers/microposts_controller_test.rb

```diff
@@ -0,0 +1,53 @@
+require "test_helper"
+
+class MicropostsControllerTest < ActionController::TestCase
+  setup do
+    @micropost = microposts(:one)
+  end
+
+  test "should get index" do
+    get :index
+    assert_response :success
+    assert_not_nil assigns(:microposts)
+  end
+
+  test "should get new" do
+    get :new
+    assert_response :success
+  end
+
+  test "should create micropost" do
+    assert_difference("Micropost.count") do
+      post  :create,
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.